testsuite/features/min_osimage_build_image.feature

```diff
@@ -20,8 +20,8 @@ Feature: Build OS images
 
   Scenario: Check the OS image built as Kiwi image administrator
     Given I am on the Systems overview page of this "sle-minion"
-    When I wait at most 2700 seconds until event "Image Build suse_os_image scheduled by kiwikiwi" is completed
-    And I wait at most 900 seconds until event "Image Inspect 1//suse_os_image:latest scheduled by kiwikiwi" is completed
+    When I wait at most 3300 seconds until event "Image Build suse_os_image scheduled by kiwikiwi" is completed
```

Comment:
create an issue for removing this seconds and replace them with wait for event

( even for later.)

Reply:
Look again at code please :cat: